1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,160 @@
-// Write your code here
+// IMAGE DATA
+
+let 
+images = [
+    {
+            src: "images/Rampart.jpg", 
+            alt: "Rampart's Landscape"
+        }, 
+        {
+            src: "images/Teldrassil.jpg", 
+            alt: "Teldrassil's Landscape"
+        }, 
+        {
+            src: "images/Chult.jpg", 
+            alt: "Chult's Landsape"
+        },
+        {
+            src: "images/Fable.jpg", 
+            alt: "Fable's Landscape"
+        },
+        {
+            src: "images/Skyrim.jpg", 
+            alt: "Skyrim's Landscape"
+        }
+    ],
+    count = 0;
+
+    
+    // CREATE SIMPLE SITE FRAME
+    
+    // CREATE NODES
+    
+    let 
+    mainDiv = document.createElement("div"),
+    heading = document.createElement("h1"),
+    currentImg = document.createElement("img"),
+    counter = document.createElement("p"),
+    buttons = document.createElement("nav"),
+    
+    autoBackBtn = document.createElement("btn"),
+    backBtn = document.createElement("btn"),
+    stopBtn = document.createElement("btn"),
+    forwardBtn = document.createElement("btn"),
+    autoForwardBtn = document.createElement("btn");
+    
+    const
+    GREY = "#3e4444",
+    LIGHT_GREEN = "#82b74b",
+    DARK_GREEN = "#405d27",
+    BROWN = "#c1946a";
+    
+    
+    // APPEND NODES
+    
+    document.body.append(mainDiv);
+    
+    mainDiv.append(heading, currentImg, counter, buttons);
+    
+    buttons.append(autoBackBtn, backBtn, stopBtn, forwardBtn, autoForwardBtn);
+    
+    
+    // RESET PRESET MARGINS
+    let allInBody = document.querySelectorAll("body, body *");
+    for (let node of allInBody) {
+        node.style.margin = "0px"; 
+    }
+
+    
+    // SET ATTRIBUTES AND STYLES
+
+mainDiv.style.textAlign = "center";
```

Comment:
Hmmm 🤔 Could you please explain what you mean by static styling?

Reply:
Dynamic styling is when you apply inline style properties using Javascript, and this is typically used to create unique styles that are not known in advance. For example, if you are allowing the user to select the color of a component.

In your case, since the styling is known in advance and is not taken as part of the user input, you should use standard CSS styles (static styles), which are much faster to load and easier to code.

Reply:
Ohh. I see. Thanks for the explanation. That makes sense! :slightly_smiling_face: